src/Reflow/Parser.hs

```diff
@@ -27,7 +26,7 @@ header = do
     n <- headerName
     s <- singleLine
     r <- many headerContinued
-    let v = s <> T.unlines r
+    let v = s <> foldl mappend "" r
```

Comment:
`foldl mappend` with the empty case of a monoid is `mconcat`!

    let v = s <> mconcat r

OR you can use `concat` from Data.Text.